src/harden.yml

```diff
@@ -15,15 +15,61 @@
         - https://raw.githubusercontent.com/cisagov/ansible-role-banner/develop/files/issue
         - https://raw.githubusercontent.com/cisagov/ansible-role-banner/develop/files/motd
     - name: Harden system
-      # This role is forked from konstruktoid/ansible-role-hardening
-      # and we do not control the names of the role variables.  This
+      # We do not control the names of the role variables.  This
       # is the reason for the noqa comment.
-      ansible.builtin.include_role:  # noqa var-naming[no-role-prefix]
-        name: harden
+      ansible.builtin.import_role:  # noqa var-naming[no-role-prefix]
+        name: konstruktoid.hardening
       vars:
-        # Point the role to the correct issue and motd templates
+        automatic_updates: true
```

Comment:
Does [konstruktoid/ansible-role-hardening](https://github.com/konstruktoid/ansible-role-hardening) definitely do the same thing as our [cisagov/ansible-role-automated-security-updates](https://github.com/cisagov/ansible-role-automated-security-updates) role?

Reply:
We turn on automated security updates for many of our AMIs, even those that are not hardened.  To remain consistent, perhaps we should continue to use our [cisagov/ansible-role-automated-security-updates](https://github.com/cisagov/ansible-role-automated-security-updates) even when using the [konstruktoid/ansible-role-hardening](https://github.com/konstruktoid/ansible-role-hardening) role.

Reply:
Will set that to `false`, since it doesn't do exactly the same thing when used on the Debian-family.
My role ensures the `APT::Periodic` are set, while https://github.com/cisagov/ansible-role-automated-security-updates ensures only security updates are installed. The settings are all default when installing `unattended-upgrades`.

(and I need to ensure that `unattended-upgrades` is installed...)